reflex/constants/installer.py

```diff
@@ -95,8 +95,8 @@ class Commands(SimpleNamespace):
         """The commands to define in package.json."""
 
         DEV = "next dev"
-        EXPORT = "next build && next export -o _static"
-        EXPORT_SITEMAP = "next build && next-sitemap && next export -o _static"
+        EXPORT = "next build"
+        EXPORT_SITEMAP = "next build && next-sitemap"
```

Comment:
The next-sitemap may have to come first to make sure it gets included in the export